test/androidTest/java/org/thoughtcrime/securesms/espresso/ConversationHelper.java

```diff
@@ -131,11 +131,11 @@ public PinnedHelper goPinned() {
         return new PinnedHelper(new HelperSecret());
     }
 
-    public PreferencesHelper goSettings() {
+    public ConversationSettingsHelper goSettings() {
```

Comment:
This might break other tests, @g-harel can you take a look

Reply:
I'll double check, but I didn't see any issues.